### ISSUE TITLE
{transpose} direction for sharps/flats not reflected in %{key_actual}

### ISSUE THREAD

**Issue by gwyndaf** [ChordPro/chordpro#298]
With 6.020, 
```
{title: Sharp/Flat transposition}
{key: C}
{transpose: -2}
The [C]chord
Key_actual: %{key_actual|%{}|"N/A"}
```
returns `A#` for `%{key_actual}`, not `Bb` as expected.

Chords themselves transpose as expected, as does `%{key}`

I recall there was some discussion of a 'smart transpose' feature (to what seems musically sensible). Not sure if this is related.

**Comment by sciurius** [comment]
Thanks, fixed in 6.020_002.

**Comment by gwyndaf** [comment]
Great. Now tests out as expected.
